tests/mr/device/mr_ref_tests.cpp

```diff
@@ -32,6 +32,7 @@ INSTANTIATE_TEST_SUITE_P(ResourceTests,
                                            mr_factory{"CUDA_Async", &make_cuda_async},
 #endif
                                            mr_factory{"Managed", &make_managed},
+                                           mr_factory{"System", &make_system},
```

Comment:
I guess this will cause the tests to fail on systems that don't support pageable memory access?

Reply:
Tests are skipped if SAM is not supported.